test/e2e/create_test.go

```diff
@@ -354,14 +354,16 @@ func (tc *testContext) waitForWindowsMachines(machineCount int, phase string, ig
 	machineStateTimeLimit := time.Minute * 5
 	startTime := time.Now()
 	// Increasing the time limit due to https://bugzilla.redhat.com/show_bug.cgi?id=1936556
-	if tc.CloudProvider.GetType() == config.VSpherePlatformType {
+	if tc.CloudProvider.GetType() == config.VSpherePlatformType ||
+		tc.CloudProvider.GetType() == config.NutanixPlatformType {
```

Comment:
Do you really need this for Nutanix also?

Reply:
Seems not necessary for Nutanix, removed Nutanix here.